client/src/components/Profile/Profile.js

```diff
@@ -15,18 +60,106 @@ function Profile({ id }) {
         return;
       }
       setUser(resJSON);
+      setIsMe(id === "me"); // TODO: this check should compare with the redux user state id instead
       setLoaded(true);
     })();
   }, [id]);
 
-  const renderUser = () => {
+  const renderOwnedMeetups = () => {
+    // TODO: get actual meetups if they exist
+    return (
+      <Typography>
+        You don’t own any meetups. <Link to="/create">Go create one!</Link>
+      </Typography>
+    );
+  };
+
+  const renderAttendingMeetups = () => {
+    // TODO: get actual meetups if they exist
+    return (
+      <Typography>
+        You are not attending any meetups.{" "}
+        <Link to="/">Look for one to attend!</Link>
+      </Typography>
+    );
+  };
+
+  const renderPendingMeetups = () => {
+    // TODO: get actual meetups if they exist
     return (
       <Typography>
-        Name is {user.name}, ID is {user.id}
+        You don’t have any pending requests.{" "}
+        <Link to="/">Look for a meetup to attend!</Link>
       </Typography>
     );
   };
 
+  const renderPersonalInfo = () => {
+    if (!isMe) {
+      return null;
+    }
+    return (
+      <>
+        {"contactInfo" in user && (
+          <Typography variant="p">Contact info: {user.contactInfo}</Typography>
+        )}
+        {"email" in user && (
+          <Typography variant="p">Email: {user.email}</Typography>
+        )}
+        {"location" in user && (
+          <Typography variant="p">
+            Location: lat {user.location.lat}, lon {user.location.lon}
+          </Typography>
+          // TODO: replace with sensible representation
+        )}
+        <Typography
+          variant="h4"
+          component="h3"
+          className={classes.meetupInfoTitle}
+        >
+          Your meetups:
+        </Typography>
+        <TabContext value={tabValue}>
+          <AppBar position="static">
+            <StyledTabList
+              onChange={(_, newValue) => setTabValue(newValue)}
+              indicatorColor="black"
+              variant="scrollable"
+              scrollButtons="auto"
+            >
+              <StyledTab label="Owned Meetups" value="1" />
+              <StyledTab label="Attending Meetups" value="2" />
+              <StyledTab label="Pending Meetups" value="3" />
+            </StyledTabList>
+          </AppBar>
+          <StyledTabPanel value="1">{renderOwnedMeetups()}</StyledTabPanel>
+          <StyledTabPanel value="2">{renderAttendingMeetups()}</StyledTabPanel>
+          <StyledTabPanel value="3">{renderPendingMeetups()}</StyledTabPanel>
+        </TabContext>
+      </>
+    );
+  };
+
+  const renderUser = () => {
+    return (
+      <Container maxWidth="md">
+        <Box display="flex" flexDirection="column" alignItems="center">
+          {"profilePic" in user && user.profilePic !== "" && (
```

Comment:
```suggestion
          {user.profilePic && (
```

But that aside, can we have a fixed "silhouette" profile pic for users without a picture stored on the server side?